src/main/java/com/ss/ita/rozetka/ui/Modals/HamburgerModal.java

```diff
@@ -0,0 +1,15 @@
+package com.ss.ita.rozetka.ui.Modals;
+
+import com.codeborne.selenide.Condition;
+import com.ss.ita.rozetka.ui.pages.*;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class HamburgerModal {
+
+    public Header closeHamburgerModal() {
+        $x("//button[@class='side-menu__close']").click();
+        $x("//nav[@class='drawer ng-star-inserted']").shouldNotBe(Condition.visible);
```

Comment:
Taras why U push your commits in my PR?

Like I know we use testng assertion, don't we?